cypress/integration/recieve-notice/recieve-notice-after-checkout.spec.js

```diff
@@ -0,0 +1,133 @@
+import uuid from 'uuid';
+import moment from 'moment';
+import checkout from '../../support/fragments/checkout/checkout';
+import searchPane from '../../support/fragments/circulation-log/searchPane';
+import circulationRules from '../../support/fragments/circulation/circulation-rules';
+import noticePolicy, { NOTICE_ACTIONS } from '../../support/fragments/circulation/notice-policy';
+import noticePolicyTemplate, { TEMPLATE_CATEGORIES } from '../../support/fragments/circulation/notice-policy-template';
+import patronGroups from '../../support/fragments/settings/users/patronGroups';
+import topMenu from '../../support/fragments/topMenu';
+import generateItemBarcode from '../../support/utils/generateItemBarcode';
+import getRandomPostfix from '../../support/utils/stringTools';
+import testTypes from '../../support/dictionary/testTypes';
+import devTeams from '../../support/dictionary/devTeams';
+import checkInActions from '../../support/fragments/check-in-actions/checkInActions';
+
+// TODO email checking
+describe('Recieving notice: Checkout', () => {
+  const ITEM_BARCODE = generateItemBarcode();
+  const patronGroup = {};
+  const userData = {
+    username: `Test_user_${getRandomPostfix()}`,
+    active: true,
+    barcode: uuid(),
+    personal: {
+      preferredContactTypeId: '002',
+      lastName: `Test user ${getRandomPostfix()}`,
+      email: 'test@folio.org',
+    },
+    departments: []
+  };
+  let templateId;
```

Comment:
you can  create empty object testData and pack all required test parameters into it